### ISSUE TITLE
Pause tasks with prompts should not require a name

### ISSUE THREAD

**Issue by brandonkal** [willthames/ansible-lint#358]
# Issue Type
- Feature request

# Ansible and Ansible Lint details

```
ansible --version
ansible 2.6.1
python 3.7.0
ansible-lint --version
3.4.23
```

- ansible installation method: pip3
- ansible-lint installation method: pip3

# Desired Behaviour

The "All tasks should be named" rule should not trigger for pause tasks with a prompt. The prompt specifies the reason for the pause task and adding that to the name would result in displaying duplicate information. This is a case where using the name attribute would be excessive.

**Comment by willthames** [comment]
The prompt and the name provide two separate functions, in my view. 

The prompt requests action by the user ("Press a key", "Enter this value") but the name describes why a pause might be required "Wait 10 seconds because IAM pretends to have created a role before it has".

While you could use prompt instead of name, I'd argue that the two are sufficiently different that I wouldn't want to relax the requirements for a name here.
